Topics/User_Experience/navigation design in complex websites.md

```diff
@@ -0,0 +1,64 @@
+## What is Navigation Design ?
+Navigation Design, also referred to as menu design, helps users to easily move through the pages of your website. It utilizes design techniques to allow ease of navigation and provide the user a better sure experience. 
+
+
+## Good vs Bad Navigation Design 
+Good navigation design allows for ease of  finding information, is intuitive and easy to use. On the other hand, bad navigation design can overwhelm the user and make finding important content difficult.  As a result,  it is important to keep the user in mind when designing the navigation of a website, especially complex websites with a multitude of pages. Reasoning about how the user can navigate through the many pages of a website, can help lay the foundation to good navigation design. 
+
+Consider using the following steps to start thinking of good navigation design  : 
+
+1. Layout all the pages on your website whether on paper of through a design tool such as Figma 
+2.  Find the relevant ways a user may navigate through those pages 
+    1. For example, home page to sign in back to home page 
+3. Determine which pages they user may need to reference most often
```

Comment:
I believe there is a typo here - it should be "the" instead of "they".

Reply:
Great catch again ! Thanks for pointing these out.